packages/web/src/common/store/pages/signon/sagas.js

```diff
@@ -742,7 +742,6 @@ function* watchConfigureMetaMask() {
 function* watchFollowArtists() {
   while (
     yield all([
-      take(signOnActions.SIGN_UP_SUCCEEDED),
```

Comment:
interesting, so simply removing this allows the fetch to continue, that right?

Reply:
yeah this was redundant because we call fetchAccountAsync after putting `SIGN_UP_SUCCEEDED` https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/src/common/store/pages/signon/sagas.js#L499-L501.

Reply:
There are def other ways to fix this though...  We could remove the watch for `accountActions.fetchAccountSucceeded.type` here as well and just call `signOnActions.FOLLOW_ARTISTS` at the end of the sign up saga after `fetchAccountAsync` https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/src/common/store/pages/signon/sagas.js#L499-L501. Or we could add a watcher for `signOnActions.signUpSucceededWithId(userId)` (called [here](https://github.com/AudiusProject/audius-protocol/blob/main/packages/web/src/common/store/pages/signon/sagas.js#L499-L501)) that sets the user id in the account store so that we don't have to wait to fetch the full account before doing the follows.

Reply:
I think this is fine for now cause with the new sign up flow pretty much all of this will go away

Reply:
Ooh gotcha I thought all the sagas were staying the same and just the Ui was changing?

Reply:
True but some of these sagas are not needed. For instance this fetch artists saga can just be an Audius-query thing inside the relevant components